### PR TITLE
Release candidate version 3.4.0

### PR DESCRIPTION
**Pull request by xamanu**
Dear @hotosm/software-testing,

We are ready and excited for the next release of the Tasking Manager (version 3.4.0). Before we publish this and install it on the HOT-TM  (https://tasks.hotosm.org) we would like to ask you to have a look at the new version and check whether everything is working fine.

For this we have installed the release candidate on the “staging instance” of the Tasking Manager (https://tasks-stage.hotosm.org). You can login as usual, but be aware that all changes you are doing on this installation, are not going to be present in the main Tasking Manager. Feel free to play around with everything, make sure everything works you usually do, and particularly have an eye on the changes and improvements we introduced:

#  Description of changes

*… for all users*

1. You should be able to select the Korean language for browsing the Tasking Manager.
2. The "What is new" page has been removed.
3. Users that haven't added an email address can no longer map nor validate until they add their email address.
4. Depending on the level of experience users now have a coloured reward icon and motivating messages on the front page and user profile to continue mapping and validating.
5. Advanced users and validators can inspect tasks in OSMCha by clicking on the button "View in OSMCha" in the advanced task information area.
6. When the "expert mode" in a user's profile has been activated, users can hover over tasks' time information to view it in full detail and clicking on it copies it to the clipboard.
7. Percentages of completeness, mapped, invalidated and bad imagery tasks on projects should be accurate and consistent.

*… for project managers*

1. As per version 3.3.2., projects can be edited by the project owner and admins. Now, the project ownership can be transferred to another user with project manager role. This can be done by editing a projects and choose the "Actions" tab.
2. You can enforce random task selection for a project. The functionality is found when editing a project under the "Settings" tab and applies to all users but not admins and project managers.
3. Validators in expert mode can classify tasks and report mapping issues based on categories, which admins and project managers can define for the whole instance - see "Manage issue" in the main drop down menu.

## Changelog

* Add Korean language (https://github.com/hotosm/tasking-manager/pull/1713).
* Remove "What is new" page (https://github.com/hotosm/tasking-manager/pull/1634)
* Don't allow mapping nor validation without email being set (https://github.com/hotosm/tasking-manager/issues/1435).
* Introduce mapper-level icons and encouragement messages (https://github.com/hotosm/tasking-manager/issues/1381).
* Functionality to transfer a project to a new owner (https://github.com/hotosm/tasking-manager/issues/1341).
* Allow to transfer only to owners with the role of a project manager (https://github.com/hotosm/tasking-manager/issues/1725).
* Enforce Random Task Selection (https://github.com/hotosm/tasking-manager/issues/1583).
* Allow advanced users and validators to inspect tasks through OSMCha (https://github.com/hotosm/tasking-manager/issues/1362).
* Show users in expert mode ISO8601 timestamps on tasks (https://github.com/hotosm/tasking-manager/issues/1359).
* Allow validators in expert mode to classify mapping issues (https://github.com/hotosm/tasking-manager/pull/1276).
* Repair calculation about on tasks completion on projects (https://github.com/hotosm/tasking-manager/issues/1527, https://github.com/hotosm/tasking-manager/pull/1696).
* Improve time-spend-mapping calculations (https://github.com/hotosm/tasking-manager/pull/1571, https://github.com/hotosm/tasking-manager/issues/1562).
* Add Task Annotations API with markdown support (https://github.com/hotosm/tasking-manager/pull/1349, https://github.com/hotosm/tasking-manager/pull/1264, https://github.com/hotosm/tasking-manager/pull/1700).
* Fix search by project number through database trigger for indexing text_searchable (https://github.com/hotosm/tasking-manager/issues/1723)
* Use textual replacements in SQL queries (https://github.com/hotosm/tasking-manager/issues/1731)
* Improve handling of strings for translations (https://github.com/hotosm/tasking-manager/issues/1678, https://github.com/hotosm/tasking-manager/pull/1706).
* Updated technical documentation (https://github.com/hotosm/tasking-manager/pull/1674, https://github.com/hotosm/tasking-manager/pull/1489).

In case you find any problems, feel free to reach out to us on Slack (#tasking-manager) or comment below.

Many thanks!

### CODE REVIEW

**Comment by russdeffner** [comment]
I would change _for project managers..._ 1. to...

'Projects can now only be edited by the Project Owner and Admins, Project ownership can be transferred to another user with project manager role. This can be done by editing a projects and choose the "Actions" tab.'

**Comment by russdeffner** [comment]
Would reword _... for all users_ 3. to...

'Users that haven't added an email address can no longer map nor validate until they add their email address.'

**Comment by mataharimhairi** [comment]
testing out the new version (which is amazing btw) and it seems that i can inspect tasks in OSMCha even though I don't have the 'expert mode' turned on? my expectation is that this wouldn't be available. this was tested with https://tasks-stage.hotosm.org/project/6175.  most of the testing was carried out in project 6175 and was pretty straight forward, except for adding notes as a validator in 'expert mode'. 

it seems i can select a category for issues under the 'Note Mapping Issues (optional)' section by using the spacebar, but then the next section 'Done editing? Leave a comment and select one of the options below that matches your editing status.' isn't followed by an option to select? Is this referring to the 'Note Mapping Issues (optional)' section above?

**Comment by xamanu** [comment]
>  it seems that i can inspect tasks in OSMCha even though I don't have the 'expert mode' turned on

Right. Thanks for spotting that. The "View OSMCha" button is actually available for validators and advanced mappers. No need to activate the expert mode. I adjusted the description above.

> it seems i can select a category for issues under the 'Note Mapping Issues (optional)' section by using the spacebar, but then the next section 'Done editing? Leave a comment and select one of the options below that matches your editing status.' isn't followed by an option to select? Is this referring to the 'Note Mapping Issues (optional)' section above?

No, the comment refers to the whole validation activity. it is not related to mapping issues.

**Comment by mataharimhairi** [comment]
> > it seems that i can inspect tasks in OSMCha even though I don't have the 'expert mode' turned on
> 
> Right. Thanks for spotting that. The "View OSMCha" button is actually available for validators and advanced mappers. No need to activate the expert mode. I adjusted the description above.
> 
> > [..] section 'Done editing? Leave a comment and select one of the options below that matches your editing status.' isn't followed by an option to select? Is this referring to the 'Note Mapping Issues (optional)' section above?
> 
> No, the comment refers to the whole validation activity. it is not related to mapping issues. Do you think we do better by separating these two sections more?

No need to separate, just make it clearer and in line with the directions when it says 'select one of the options below that matches your editing status.' - then there should be an option below, not above :)

**Comment by xamanu** [comment]
Yes, this is indeed very misleading. In order to solve this I suggest to change the wording to `Done editing? Share a comment and mark the task with one of the buttons below.` -> https://github.com/hotosm/tasking-manager/pull/1754
